jha/src/store/states/enrolment-module-dummy-data.js

```diff
@@ -30,23 +30,24 @@ export default {
   ahCitizenshipStatusReason: CanadianStatusReasons.LivingInBCWithoutMSP,
   ahCitizenshipSupportDocumentType: SupportDocumentTypes.CanadianBirthCertificate,
   ahCitizenshipSupportDocuments: [{}],
-  ahIsNameChanged: 'N',
-  ahNameChangeSupportDocumentType: null,
+  ahIsNameChanged: 'Y',
+  ahNameChangeSupportDocumentType: "Marriage Certificate",
   ahNameChangeSupportDocuments: [],
```

Comment:
If you're adding in a name change, can you add an empty object to the support documents? This will prevent us from having to manually upload a doc when using the dummy data (except for when submitting).

ie:
```javascript
ahNameChangeSupportDocuments: [{}],
```

Reply:
Ooh good catch! I did this